documentation/modules/ROOT/pages/_partials/k8s-build-deploy.adoc

```diff
@@ -2,47 +2,48 @@
 
 #tag::k8s-prep[]
 
-Copy the pass:[$TUTORIAL_HOME]/templates/app.yaml to `{quarkus-project-name}` Quarkus Application directory:
+[IMPORTANT]
+====
+If you have not deployed the application previously, you need to add Quarkus Kubernetes extension. This extension uses https://github.com/dekorateio/dekorate[Dekorate] to generate a default Kubernetes resource template.
 
-[#conk8s-copy-k8s-app-template]
+[#qext-mvn-add-kubernetes-extension]
 [source,bash,subs="+macros,+attributes"]
 ----
-cp pass:[$TUTORIAL_HOME]/templates/app.yaml .
+./mvnw quarkus:add-extension -Dextensions="quarkus-kubernetes"
```

Comment:
since we will use dekorate, shall we have the app.yaml from the template project removed?

Reply:
yes